scripts/image-mounter.mjs

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env node
+/**
+ * Image mounter script for mounting and unmounting Personalized Developer Disk Image on iOS devices
+ * This script provides a CLI interface to mount/unmount DDI via remote XPC services
+ */
+
+import {logger} from '@appium/support';
+import {promises as fs} from 'fs';
+import path from 'path';
+import {Command} from 'commander';
+
+const log = logger.getLogger('ImageMounter');
+
+/**
+ * ImageMounter class for managing image mounting and unmounting operations
+ */
+class ImageMounter {
+  constructor() {
+    this.remoteXPCModule = null;
+  }
+
+  async initializeRemoteXPC() {
+    if (this.remoteXPCModule) {
+      return this.remoteXPCModule;
+    }
+
+    try {
+      this.remoteXPCModule = await import('appium-ios-remotexpc');
+      return this.remoteXPCModule;
+    } catch {
+      throw new Error(
+        'appium-ios-remotexpc is not installed. Please install it using:\n' +
+        'npm install appium-ios-remotexpc\n\n' +
+        'Note: This is an optional dependency required for image mounting functionality.'
+      );
+    }
+  }
+
+  async validateFile(filePath, fileType) {
+    try {
+      const absolutePath = path.resolve(filePath);
+      const stat = await fs.stat(absolutePath);
+
+      if (!stat.isFile()) {
+        throw new Error(`${fileType} path is not a file: ${absolutePath}`);
+      }
+
+      return absolutePath;
+    } catch (error) {
+      if (error.code === 'ENOENT') {
+        throw new Error(`${fileType} file not found: ${path.resolve(filePath)}`);
+      }
+      throw error;
+    }
+  }
+
+  async getTargetDevice(udid) {
+    const remoteXPC = await this.initializeRemoteXPC();
+    const {createUsbmux} = remoteXPC;
+
+    log.info('Connecting to usbmuxd...');
+    const usbmux = await createUsbmux();
+
+    const devices = await usbmux.listDevices();
+    if (devices.length === 0) {
+      await usbmux.close();
+      throw new Error('No devices found. Make sure iOS devices are connected and trusted.');
```

Comment:
Maybe here can suggest https://appium.github.io/appium-xcuitest-driver/latest/preparation/real-device-config/ also. I don't recall but maybe developer mode would be also needed...?